src/main/java/org/stellar/sdk/ManageOfferOperation.java

```diff
@@ -55,7 +56,7 @@ public String getAmount() {
   /**
    * Price of 1 unit of selling in terms of buying.
    */
-  public String getPrice() {
+  public Price getPrice() {
```

Comment:
Can we make this a non-breaking change? The new method can be: `public Price getPriceObject()` (let me know if you have a better name idea).